static/library-scripts/azcli-debian.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
```

Comment:
Just double-checking... This file is taken as-is from the MS repo, correct? No modifications?

Reply:
Iirc yes

Reply:
rien changé ici non plus